tests/test_synthetic_data.py

```diff
@@ -126,18 +131,35 @@ def _single_trajectory_integration(
     position_scalars = [f"traj_tracer_{s}" for s in ["xr", "xi", "yr", "yi", "zr"]]
     ds_position_scalars = ds[position_scalars]
 
+    nrep = 3
```

Comment:
thanks for extending this to having multiple trajectories! I think it would be good to rename the test though, or simply have two tests 1) one with only one trajectory and 2) one with multiple. What do you think?

Reply:
Good idea. I'll implement and test.

Reply:
I've renamed and implemented both single and multiple tests in teh same function, because it's called a lot of times I didn't want to double up all the calls.